README.MD

```diff
@@ -74,22 +115,22 @@ pm.register(minLength, maxLength);
 
 const result = pm.validate('MyPassword123!*') // Throws PasswordValidatorConflictException --> `minLength cannot be greater than maxLength`
```

Comment:
Conflict checks and errors happen at the point of registering the validators.

`pm.register(minLength, maxLength);`

and not at the point of validating the password. Please fix it.

Reply:
Nice catch! Thanks